src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,9 +131,10 @@ private void initSubsystems() {
           case "ClimbSubsystem":
             {
               System.out.println("Climb enabled");
-              climbSubsystem = new ClimbSubsystem();
-              climbUPCommand = new ClimbUPCommand(climbSubsystem);
-              climbDOWNCommand = new ClimbDOWNCommand(climbSubsystem);
+              if (!Constants.oneController) {
+                climbSubsystem = new ClimbSubsystem(operatorJoystick);
```

Comment:
Because we only have a button mapping for this if there are two controllers, only make this enabled if that second controller is used